example/30-cloudprofile-alicloud.yaml

```diff
@@ -15,9 +15,9 @@ spec:
       - name: unmanaged
       kubernetes:
         versions:
-        - 1.12.1
-        - 1.11.3
-        - 1.10.8
+        - 1.12.3
+        - 1.11.5
+        - 1.10.11
```

Comment:
is there a reason why the `# - 1.9.11 # Usage is not recommended, see CVE-2018-1002105` was not added here as well?

Reply:
+1

Reply:
The reason is that 1.9.x has never been part of the Alicloud Cloudprofile and I didn't want to question that in this PR. I can still add it, though.

Reply:
I would add it for the sake of uniformity